api/src/main/java/org/pmiops/workbench/api/AuditController.java

```diff
@@ -127,12 +124,6 @@ private static String auditSql(String cdrProjectId, List<String> tableSuffixes)
                       + "project jobUser",
                   cdrProjectId, total, email));
           numBad += total;
-        } else if (!whitelist.contains(project_id)) {
```

Comment:
@calbach this is the one thing I'm not confident with. We don't have whitelist anymore cause we don't have user projects. Is there another way to do this, or is it good to remove?

Reply:
Should I just get rid of the conditions and log all bad actors here?

Reply:
The equivalent replacement here would be to query the workspace dao for all workspace namespaces